File: src/api/LMSEntities/DataTransferObjects/CheckoutForDetailedDto.cs
using System;

namespace LMSEntities.DataTransferObjects
{
    public class CheckoutForDetailedDto
    {
        public int Id { get; set; }
        public LibraryAssetForListDto LibraryAsset { get; set; }
        public LibraryCardForDetailedDto LibraryCard { get; set; }
        public DateTime CheckoutDate { get; set; }
        public DateTime DueDate { get; set; }
        public string Status { get; set; }
        public DateTime? DateReturned { get; set; }
    }
}

File: src/spa/src/dto/models/checkout-for-detailed-dto.ts
/* tslint:disable */
/* eslint-disable */
import { LibraryAssetForListDto } from './library-asset-for-list-dto';
import { LibraryCardForDetailedDto } from './library-card-for-detailed-dto';
export interface CheckoutForDetailedDto {
  checkoutDate?: string;
  dateReturned?: null | string;
  dueDate?: string;
  id?: number;
  libraryAsset?: LibraryAssetForListDto;
  libraryCard?: LibraryCardForDetailedDto;
  status?: null | string;
}
